web/src/js/components/about/AboutContainer.js

```diff
@@ -14,42 +14,71 @@
  * limitations under the License.
  */
 
-import React from "react";
-
+import React from 'react';
```

Comment:
It seems elements of this PR are still related to a react change and not adding eslint?

Reply:
I think this change is shown due to the eslint rule for using consistent quotes. So the change here is the modification from double quotes to single quotes.